working/augmentation-libraries/feature-specification.md

```diff
@@ -738,85 +782,52 @@ It's a **compile-time** error if:
     that can be abstract. In that case, if no declaration provides a body, it
     is considered abstract.*
 
-### Augmenting variables
-
-A class-like augmentation can add *new* variables (that is instance or static
-fields) to the type being augmented:
+### Augmenting variables, getters, and setters
 
-```dart
-class C {}
-
-augment class C {
-  int x = 3;
-
-  static int y = 4;
-}
-```
+For purposes of augmentation, a variable declaration is treated as implicitly
+defining a getter whose return type is the type of the variable. If the variable
+is not `final` or is `late`, then the variable declaration also implicitly
```

Comment:
", or is `late` and does not have an initializer expression"

Reply:
Ha, I knew I was going to get an edge case wrong here.